app/src/main/res/values/strings_settings.xml

```diff
@@ -30,81 +30,84 @@
+    <string name="hint_folders_number">Create as many folders as you want to organize your apps.</string>
+    <string name="hint_folder_edit">Click on a folder name to change it. The pen allows editing folder content.</string>
     <string name="hint_new_folder">New folder</string>
     <string name="hint_rename_folder">Rename folder</string>
-    <string name="hint_remove_folder">If a folder is removed, its content will be put back in the main applications list.</string>
-    <string name="error_folder_empty_name">The name cannot be empty!</string>
-    <string name="error_folder_already_exists">This folder already exists!</string>
-    <string name="error_folder_rename">An error happened while renaming the folder!</string>
+    <string name="hint_remove_folder">If a folder is removed, its content will be put back in the main app list.</string>
+    <string name="error_folder_empty_name">Pick a name first</string>
+    <string name="error_folder_already_exists">Pick a name that doesn\'t exist already</string>
+    <string name="error_folder_rename">Could not rename the folder</string>
 
     <!-- Shortcuts -->
-    <string name="error_shortcut">An error happened with the shortcut %1$s!</string>
-    <string name="error_shortcut_invalid_request">The received request to add a shortcut is not valid!</string>
-    <string name="error_shortcut_missing_info">Some information are missing and the shortcut cannot be launched!</string>
-    <string name="error_shortcut_not_default_launcher">Only the default launcher can open shortcuts!</string>
-    <string name="error_shortcut_start">The Android system was not able to open the shortcut or it did not retrieved it!</string>
+    <string name="error_shortcut">Could not follow the %1$s shortcut</string>
+    <string name="error_shortcut_invalid_request">Could not add that as a shortcut</string>
+    <string name="error_shortcut_missing_info">Could not follow this shortcut</string>
+    <string name="error_shortcut_not_default_launcher">Only the default launcher can open shortcuts</string>
+    <string name="error_shortcut_start">The Android system could not open the shortcut</string>
 
     <!-- Search -->
     <string name="search_icon_title">Search</string>
-    <string name="search_hint">Search applications</string>
+    <string name="search_hint">Search for apps</string>
 
     <!-- Other settings -->
     <string name="set_transparent_status_bar">Transparent status bar</string>
     <string name="set_transparent_status_bar_hint">When favorites are closed</string>
-    <string name="set_hide_menu_button">Hide the menu button</string>
-    <string name="set_hide_menu_button_help">It stays accessible using the launcher icon</string>
+    <string name="set_hide_menu_button">Hide the 🔧 menu button</string>
+
+    <string name="set_hide_menu_button_help">Always accessible from the 🔧 menu button and the launcher app icon</string>

```

Comment:
```suggestion
    <string name="set_hide_menu_button_help">Always accessible from the launcher app icon</string>
```
It should be reverted because the purpose of the related setting (on line 74) is to hide the 🔧 menu button (so it will not be accessible anymore when the setting is enabled).
The comment about the launcher app icon is to let the user know that (s)he will not be locked out of the menu when enabling this setting.